src/community/Community.js

```diff
@@ -9,9 +15,63 @@ class Community extends Component {
     return (
       <div className="community-page container">
         <DocumentTitle title="Our community | #NYCEDU" />
+        <object id="sky-background" className="community-backgrounds" type="image/svg+xml" data={sky}>Your browser does not support svgs.</object>
         <h1>Community</h1>
         <p>If you care about education in NYC, then you are already a member of #NYCEDU! Meet some of your fellow community members.</p>
-        <SignupForm handleSubmission={this.props.handleToast} />
+        <div className="member-container">
```

Comment:
Create a data structure to hold the community members, then .map over it to generate the markup.